101-asp-net-app-service/GettingStarted.md

```diff
@@ -0,0 +1,159 @@
+You gonna be redirected to the Azure Portal where you can use your credentials to login into.
+
+After login, you gonna have your credentials. 
+
+In order to set the right subscription, you can use the follow command:
+
+```
+az account set --subscription "< your subscription id >"
+```
+
+### Resource Group
+
+A resource group is a container that holds related resources for an Azure solution. The resource group includes those resources that you want to manage as a group. 
+
+We gonna need to create a resource group for our deployment if we haven't yet create a resource group.
+
+To create a resource group, we will need a name and the location. For a list of locations, type:
+
+```
+az account list-locations
+```
+
+To create the resource group, just type the command:
+
+```
+az group create --name <mygroupname> --location <thelocation>
+```
+
+![Screen](./images/az-groupcreate.png)
+
+Now, we are ready to deploy our template. Type the next command to get it:
+
+```
+az group deployment create --resource-group <your resource-group name> --template-file <full path and name of your template>
+```
+
+![Screen](./images/az-group-deploy.png)
+
+It will spend some time on the deployment. Bring it like 5 minutes until the process finish.
+
+When it finished, let's go to the Portal, and let's see our App Services.
+
+![Screen](./images/portal-resource.png)
+
+Congratulations! You have deployed the template succesfully. We can see our web visiting an url like that:
+
+```
+<app-name>.azurewebsites.net
+```
+
+You can redeploy it automatically with just click on this button: 
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-asp-net-app-service%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-asp-net-app-service%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png"/>
+</a>
+
+
+### Deploy your app
+
+To deploy your our ASP.NET app you can follow that [tutorial](https://docs.microsoft.com/en-in/azure/app-service/app-service-web-get-started-dotnet#create-an-aspnet-core-web-app). 
+
+Just take care when you want to publish your application, because you have to select an existing app service (the one que have created).
+
+### How to shutdown your resources
+
+On the portal, open your resource group and click on the Delete button.
+
+![Screen](./images/delete-rsc.png)
+
+### What to do if deployment fails
+
+When we are trying to deploy our template, we can find that this fails. If that happens, what we will do it is to delete the resource group and then redeploy our template.

```

Comment:
Why not just deploy the template again - shouldn't need to delete anything

Reply:
It was like that because of if the deployment fails because a name is already taken, the deployment fails and like the other resources are linked to the name, if you deploy again changing the name you will have resources that you don't need.